src/main/flight/imu.h

```diff
@@ -58,6 +58,7 @@ extern attitudeEulerAngles_t attitude;
 typedef struct imuConfig_s {
     uint16_t dcm_kp;                        // DCM filter proportional gain ( x 10000)
     uint16_t dcm_ki;                        // DCM filter integral gain ( x 10000)
+    uint8_t recovery_coef;
```

Comment:
Because this struct is used in a parameter group (see below), the new entry needs to go at the end.